app/components/developer.py

```diff
@@ -17,6 +19,21 @@ class Developer(commands.Cog):
     def __init__(self, bot: GhosttyBot) -> None:
         self.bot = bot
 
+    async def existing_extension_autocomplete(
+        self, _: dc.Interaction, current: str
+    ) -> list[Choice[str]]:
+        return sorted(
+            (
+                Choice(name=name, value=cog_module.__name__)
+                for name, cog in (c for c in self.bot.cogs.items())
```

Comment:
Correct me if I'm wrong, but the genexpr feels unnecessary, since it doesn't do any copies and is still a view of the original dict:
```suggestion
                for name, cog in self.bot.cogs.items()
```

Reply:
Oh yea, this is an artifact from when this autocomplete was different. Good catch